ArduinoController/ArduinoController.ino

```diff
@@ -10,43 +10,41 @@
 #include "StartController.h"
 #include "PinJSON.h"
 
-const uint8_t KILLPIN = 50; // Needs to remain global (therefore outside of JSON) for lambda interrupt
-const uint8_t KILL_ADDR = 10;
 const uint32_t CONTROLLER_CNT = 13u;
 class IController* controllers[CONTROLLER_CNT];
 
 void setup() {
   Serial.begin(115200);
-  Serial.print('R');
-  DynamicJsonBuffer buffer;
-  auto json = buffer.parse(PinJSON::json);
+  Serial1.begin(9600);
+  SerialTools::writeString("Ready!", 6);
   
-  controllers[0] = new ThrustController(json["thrust"]["left_forward"].as<uint8_t>(), 50);
```

Comment:
Might consider adding integer constants for these array indices in the future, but it doesn't prevent approval of this PR.